infrastructure/helm.tf

```diff
@@ -0,0 +1,79 @@
+resource "helm_release" "load_balancer_controller" {
+  name             = "stefan-loadbalancer-controller"
+  repository       = "https://aws.github.io/eks-charts"
+  chart            = "aws-load-balancer-controller"
+  namespace        = "vegait-training"
```

Comment:
lb controller should be in it's own namespace

Reply:
So, does the namespace need to be what I put in the irsa load balance module?